bucket/activemq.json

```diff
@@ -0,0 +1,69 @@
+{
+    "version": "5.16.2",
+    "description": "Apache ActiveMQ is the most popular open source, multi-protocol, Java-based message broker",
+    "homepage": "https://activemq.apache.org/",
+    "license": {
+        "identifier": "Apache License 2.0",
+        "url": "https://activemq.apache.org/what-is-the-license"
+    },
+    "notes": [
+        "Apache ActiveMQ installed as 'ActiveMQ' service.",
+        "Open 'Services (services.msc)' to configure ActiveMQ startup type."
+    ],
+    "url": "https://downloads.apache.org//activemq/5.16.2/apache-activemq-5.16.2-bin.zip",
+    "hash": "sha512:a3ca616b89d32faa73f7e3d31fe3d1d6cf789f711da3b87df14f74f102072eadfe03218d051ac76808785143782c6e60ee0ef74f0fb097affb5cd138bb1365cb",
+    "extract_to": "tmp",
+    "pre_install": [
+        "if (-not (is_admin)) { throw 'Administrator privileges are needed for installation' }"
+    ],
+    "installer": {
+        "script": [
+            "(Get-ChildItem -Directory \"$dir\\tmp\").FullName | % { Move-Item \"$_\\*\" \"$dir\" }",
+            "Remove-Item -Recurse \"$dir\\tmp\""
+        ]
+    },
+    "post_install": [
```

Comment:
Do not manipulate with users system

Reply:
Because apache include folder (ex: apache-activemq-5.16.2) inside the zip file.
I extract it to tmp folder then copying all files inside apache-activemq-5.16.2 folder to $dir folder then remove the tmp folder.